include/spirv/1.0/extinst.glsl.std.450.grammar.json

```
@@ -1,28 +1,12 @@
 {
   "copyright" : [
-    "Copyright (c) 2014-2016 The Khronos Group Inc.",
-    "",
-    "Permission is hereby granted, free of charge, to any person obtaining a copy",
-    "of this software and/or associated documentation files (the \"Materials\"),",
-    "to deal in the Materials without restriction, including without limitation",
-    "the rights to use, copy, modify, merge, publish, distribute, sublicense,",
-    "and/or sell copies of the Materials, and to permit persons to whom the",
-    "Materials are furnished to do so, subject to the following conditions:",
-    "",
-    "The above copyright notice and this permission notice shall be included in",
-    "all copies or substantial portions of the Materials.",
+    "Copyright:  2014-2016 The Khronos Group Inc.",
+    "License: MIT",
```

This same comment applies to multiple files in this PR.

Should we be using the SPDX text instead, instead of "Copyright" and "License" generally?

```suggestion
    "SPDX-FileCopyrightText:  2014-2016 The Khronos Group Inc.",
    "SPDX-License-Identifier: MIT",
```

---

Fixed.

---

My original comment has been edited regarding REUSE handling of JSON files.

---

Got it.  So in general we should use the SPDX labels, unless it's a JSON file, in which case we'll just use the general "Copyright" and "License" labels.